app/controllers/sites/scope_filters_controller.rb

```diff
@@ -0,0 +1,49 @@
+module Sites
+	class ScopeFiltersController < ApplicationController
+		include Dcv::RestrictableController
+		include Cul::Omniauth::AuthorizingController
+		before_filter :load_subsite
+		before_filter :authorize_site_update
+		layout 'sites'
+
+		def load_subsite
```

Comment:
I don't think that controller instance variable are shared across requests, so I'm not sure if you're benefitting from the `||=` going on here.  It looks like `load_subsite ` is only being called once.

Reply:
that's borrowed from a general implementation - there are definitely circumstances where it is called more than once (sometimes from views as `controller.load_subsite`).

Reply:
👍